docs/self-managed/operational-guides/configure-db-custom-headers.md

```diff
@@ -0,0 +1,152 @@
+---
+id: configure-db-custom-headers
+title: "Configure DB client custom headers"
+sidebar_label: "Configure DB client custom headers"
+description: "Learn how to configure DB client custom headers"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+:::caution
+This guide describes a feature preview and is a matter for change.
+:::
+
+Camunda supports plugin loading feature that can be attached to ElasticSearch / OpenSearch HTTP client, providing
+an ability to add user-controlled custom HTTP headers.
+
+This use case may be applicable when one needs to add custom HTTP headers to search DB call, for
+example authentication, tracking, debugging, or other cases.
+
+## Technical overview
+
+In order to introduce your custom plugin to Camunda, create a new Java project, and add the following dependency:
+
+<Tabs groupId="dependency" defaultValue="maven" values={
+[
+{label: 'Maven dependency', value: 'maven' },
+{label: 'Gradle dependency', value: 'gradle' }
+]
+}>
+
+<TabItem value='maven'>
+
+```xml
+<dependency>
+  <groupId>io.camunda</groupId>
+  <artifactId>camunda-search-plugins</artifactId>
```

Comment:
I found it confusing at first that it's "search" plugin. Do you refer to OpenSearch and Elastic or why search? Isn't it more about the database? camunda-db-plugins e.g.?